conf/application.conf

```diff
@@ -87,6 +87,13 @@ microservice {
         port = 9482
         startUrl = "transit-movements-trader-reference-data"
       }
+
+      router {
```

Comment:
Do we think there's any value in adding _test-only/testOnly_  to this service name? Just so people are aware that this connection isn't available in live?

Reply:
Good shout, will add in.